ml-package-versions-master.yml

```diff
@@ -0,0 +1,34 @@
+sklearn:
+  models:
+    package: "scikit-learn"
+    minimum: "0.19.2"
+    maximum: "0.23.2"
+    run: |
+      pytest tests/sklearn/test_sklearn_model_export.py --large
+
+  autologging:
+    package: "scikit-learn"
+    minimum: "0.22.2"
+    maximum: "0.23.2"
+    unsupported: ["0.22.2"]
```

Comment:
Note that `0.22.2` isn't actually unsupported (this was used to showcase an example of `unsupported` in the design doc).

Reply:
Removed :)